subprocess/subtree_oom_darwin.go

```diff
@@ -0,0 +1,82 @@
+// +build darwin
+
+package subprocess
+
+import (
+	"bufio"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func (o *OOMTracker) Clear() error {
+	var err error
+	o.IsSudo, err = isSudo()
+	if err != nil {
+		return errors.Wrap(err, "error checking sudo")
+	}
+
+	if o.IsSudo {
+		return errors.Wrap(exec.Command("sudo", "log", "erase", "--all").Run(), "error clearing log")
+	}
+
+	return errors.Wrap(exec.Command("log", "erase", "--all").Run(), "error clearing log")
+}
+
+func (o *OOMTracker) Check() error {
+	wasOOMKilled, pids, err := analyzeLogs(o.IsSudo)
+	if err != nil {
+		return errors.Wrap(err, "error searching log")
+	}
+	o.WasOOMKilled = wasOOMKilled
+	o.Pids = pids
+	return nil
+}
+
+func analyzeLogs(isSudo bool) (bool, []int, error) {
+	var cmd *exec.Cmd
+	wasOOMKilled := false
+	pidChannel := make(chan *int)
+
+	if isSudo {
+		cmd = exec.Command("sudo", "log show")
```

Comment:
I would have expected `("sudo", "log", "show")`. Does this form work? Is there a reason to prefer `("sudo", "log show")`

Reply:
no reason to prefer it, will switch